R/input_output_filter.R

```diff
@@ -173,7 +185,7 @@ load_2_dms = function(dm1_fp, dm2_fp, map_fp, filter_cat, filter_vals, keep_vals
 #'  the loaded mapping file, and optionally (3) the loaded taxonomy information.
 #' @examples 
 #' \dontrun{
-#' ex_in_filt = filter_data(input = "example_input", filter_cat = "Sample_type", 
+#' filter_data(input = input, filter_cat = "Sample_type", 
```

Comment:
Looks good - just change 'input' to 'fruits_veggies' since that is the example dataset that is included with the package